app/views/home_page/_call_to_action.html.erb

```diff
@@ -17,7 +17,11 @@
             Available now. Free for all UCLA students.
           </h2>
           <div class="w-full flex justify-center">
-            <%= button_to 'Sign up', user_google_oauth2_omniauth_authorize_path, data: { turbo: false }, class: 'cursor-pointer mt-6 mx-auto block rounded-md border border-transparent px-5 py-3 bg-gray-900 text-base font-medium text-white shadow hover:bg-black focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-red-500 sm:px-10' %>
+            <%= button_to 'Sign up', user_google_oauth2_omniauth_authorize_path, data: { turbo: false }, class:
```

Comment:
Are we adding a login button here? I just see the sign up button.

Reply:
I only added a login button on the header file. I tried putting one here too but it looked a bit strange so I just left it as the sign up button. I can a login button here too if needed. Alternatively, on a different ticket, we could just have the login and sign up button both on a navbar that sticks to the top?

Reply:
Could you post a screenshot of what the login button on the header file looks like?

Reply:
<img width="1210" alt="Screen Shot 2023-02-04 at 4 42 44 PM" src="https://user-images.githubusercontent.com/92414245/216795656-82ec49e7-d77c-43bc-88e6-a2c9bd8363d4.png">
<img width="1125" alt="Screen Shot 2023-02-04 at 4 42 38 PM" src="https://user-images.githubusercontent.com/92414245/216795659-479b2d88-3fbf-41a0-82ab-c482ac015f64.png">

Reply:
@lzhou0714 Sweet! Could we just flip the order of the buttons so that "Sign up" is on the right?

And for the call to action part down here, let's update the text to say "Log in / Sign up" - that's what we do on the navbar on other pages.

Reply:
<img width="1728" alt="Screen Shot 2023-02-06 at 4 38 14 PM" src="https://user-images.githubusercontent.com/92414245/217118732-36ea2be5-8ccf-43d3-801a-bcff5fd0bdf1.png">
<img width="1725" alt="Screen Shot 2023-02-06 at 4 38 25 PM" src="https://user-images.githubusercontent.com/92414245/217118749-60e48b5b-a597-4bea-aa0f-52bcee830904.png">

Reply:
@nathanmsmith I made the changes! I flipped the color and of login and signup because it looked a little strange but other than that it's basically the same thing just different order.